tests/keras/test_callbacks.py

```diff
@@ -909,5 +909,90 @@ def tests_RemoteMonitorWithJsonPayload():
                   validation_data=(X_test, y_test), callbacks=cbks, epochs=1)
 
 
+@pytest.mark.skipif((K.backend() != 'mxnet'),
+                    reason='Supported for MXNet backend only.')
+@keras_test
+def test_MXNetModelCheckpoint():
```

Comment:
please consider breaking down this down into multiple unit tests

Reply:
Done.